docs/content/engines/quick_start.md

```diff
@@ -26,43 +26,138 @@ under the License.
+the version you want
+
+### Docker compose
+
+docker compose is a convenient way to manage docker containers and quickly build the application environment 
+you need. The following docker compose file is composed of three services, jobmanager, taskmanager and sql-client
+, using the flink:1.17.1-scala_2.12 image.
+
+Copy the following into a ```flink_docker_compose.yml``` file
+
+> The /tmp path in the container is mapped to the local./tmp path so that multiple containers can share a file system path. The reason is because the 'warehouse'='file:/tmp/paimon' option is configured when the catalog is created. If you use another path as the warehouse address, modify the volumes option in the docker compose file simultaneously.
+
+```shell
+
+version: "2.2"
+services:
+  jobmanager:
+    image: flink:1.17.1-scala_2.12
+    ports:
+      - "8081:8081"
+    command: jobmanager
+    user: flink
+    environment:
+      - |
+        FLINK_PROPERTIES=
+        jobmanager.rpc.address: jobmanager
+    networks:
+      - flink-network
+    volumes:
+      - ./custom_jars/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar:/opt/flink/lib/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar
+      - ./custom_jars/paimon-flink-1.17-0.5-20230815.001819-123.jar:/opt/flink/lib/paimon-flink-1.17-0.5-20230815.001819-123.jar
+      - ./tmp:/tmp
+
+  taskmanager:
+    image: flink:1.17.1-scala_2.12
+    depends_on:
+      - jobmanager
+    command: taskmanager
+    user: flink
+    scale: 1
+    environment:
+      - |
+        FLINK_PROPERTIES=
+        jobmanager.rpc.address: jobmanager
+        taskmanager.numberOfTaskSlots: 4
+    networks:
+      - flink-network
+    volumes:
+      - ./custom_jars/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar:/opt/flink/lib/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar
+      - ./custom_jars/paimon-flink-1.17-0.5-20230815.001819-123.jar:/opt/flink/lib/paimon-flink-1.17-0.5-20230815.001819-123.jar
+      - ./tmp:/tmp
+
+  sql-client:
+    image: flink:1.17.1-scala_2.12
+    command: bin/sql-client.sh
+    user: flink
+    depends_on:
+      - jobmanager
+    environment:
+      - |
+        FLINK_PROPERTIES=
+        jobmanager.rpc.address: jobmanager
+        rest.address: jobmanager
+    networks:
+      - flink-network
+    volumes:
+      - ./custom_jars/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar:/opt/flink/lib/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar
+      - ./custom_jars/paimon-flink-1.17-0.5-20230815.001819-123.jar:/opt/flink/lib/paimon-flink-1.17-0.5-20230815.001819-123.jar
+      - ./tmp:/tmp
+
+networks:
+  flink-network:
+    driver: bridge
+
+```
 
 ### Step 1: Prepare the dependent jar files
 
-[Download](https://flink.apache.org/downloads.html) Pre-bundled Hadoop jar and copy the jar file to the lib directory of your Flink path.
+Reading [Engines/Flink](https://paimon.apache.org/docs/master/engines/flink/) chapters, Download the appropriate versions of the paimon-flink-*.jar and flink-shaded-hadoop-2-uber-*.jar files to the specified path
```

Comment:
It might be confusing for beginners. The path does not need to be under the incubator-paimon repository, right? 
Nit: Avoid using too many "the" in the document unless you have already mentioned the item you are referring to. It may make readers wonder if they miss something.

Reply:
Yes, that's useful advice.

Reply:
I seem to have submitted a change twice, I will pay attention to it next time.
https://github.com/apache/incubator-paimon/pull/1832/commits/8127f69b787357355f261d6837eb06ce966fafba